x11-libs/libva-intel-media-driver/libva-intel-media-driver-21.1.2_pre.ebuild

```diff
@@ -0,0 +1,62 @@
+# Copyright 1999-2021 Gentoo Authors
+# Distributed under the terms of the GNU General Public License v2
+
+EAPI=7
+
+inherit cmake
+
+if [[ ${PV} == *9999 ]] ; then
+	: ${EGIT_REPO_URI:="https://github.com/intel/media-driver"}
+	if [[ ${PV%9999} != "" ]] ; then
+		: ${EGIT_BRANCH:="release/${PV%.9999}"}
+	fi
+	inherit git-r3
+else
+	MY_PV="${PV%_pre}"
+	SRC_URI="https://github.com/intel/media-driver/archive/intel-media-${MY_PV}.tar.gz"
+	S="${WORKDIR}/media-driver-intel-media-${MY_PV}"
+	KEYWORDS="amd64"
```

Comment:
Don't commit as [stable](https://devmanual.gentoo.org/keywording/index.html#keywording-on-upgrades) - "ekeyword ~all foo.ebuild". repoman commit or pkgdev commit then pkgdev push should warn you about this